sssd_test_framework/roles/ipa.py

```diff
@@ -328,6 +328,44 @@ def test_example(ipa: IPA):
         """
         return IPAIDView(self, name)
 
+    def hbac(self, name: str) -> IPAHbac:
+        """
+        IPA HBAC object.
+
+        Here, we can the IPA HBAC, that can be used
+        to create the rule, set the access for perticular host
+        for perticular services
+
+        .. code-block:: python
+            :caption: Example usage
+
+            @pytest.mark.topology(KnownTopology.IPA)
+            def test_ipa__validate_hbac_rule_check_access_sshd_service(client: Client, ipa: IPA):
+                # Disable all users to access all services on all hosts.
+                ipa.hbac("allow_all").disable()
+
+                rule1 = ipa.hbac("rule1").create(description="This is a new rule1").add_user("user1").add_host("client.test").add_service("sshd")
+
+                # Run and parse hbactest
+                # user1 and user2 with valid rule1
+                hbactest_out1 = rule1.hbactest(user="user1", host="client.test", service="sshd", rule="rule1")
```

Comment:
rule1, suppose to be the name of the rule? Something more descriptive so we don't have to figure what the rule does. 

"First Rule" and in the create method "HBAC rule with user1".